Question: building without self_test.

Is it no longer possible to build without `czmq_selftest`?

I see `--without-zmakecert` has been changed to `--disable-zmakecert` but `--without-czmq_selftest` has been removed.

Related to the above question is this error. Building either `czmq_selftest` or `zmakecert` results in this linker error on my Ubuntu build. I've been avoiding this by not building these, but without that option...
```
  CC       src/src_zmakecert-zmakecert.o
  CC       src/src_czmq_selftest-czmq_selftest.o
  CCLD     src/libczmq.la
  CCLD     src/czmq_selftest
  CCLD     src/zmakecert

/usr/bin/ld: /home/eric/bs-prefix//lib/libzmq.a(usrsrc_libzmq_la-/bindecoder_allocators.o)/ld:: undefined / referencehome/eric to/bs- symbolprefix/ 'ceil@@lib/libzmq.aGLIBC_2.2.5(src_libzmq_la'
/-decoder_allocators.o/lib/): undefinedx86_64- reference linux-gnuto /libm.so.6symbol 'ceil@@GLIBC_2.2.5'
//lib/x86_64: -linuxerror adding-gnu symbols/libm.so.6::  errorDSO missing adding from  command symbols:line
DSO missing from command line

collect2: error: ld returned 1 exit status
collect2: error: ld returned 1 exit status
make[1]: *** [src/zmakecert] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [src/czmq_selftest] Error 1
```
It appears that `libm` is either missing or improperly ordered in the dependencies for these two builds.

--- a/configure.ac
+++ b/configure.ac
@@ -392,6 +392,16 @@ AC_ARG_ENABLE([zmakecert],
 AM_CONDITIONAL([ENABLE_ZMAKECERT], [test x$enable_zmakecert != xno])
 AM_COND_IF([ENABLE_ZMAKECERT], [AC_MSG_NOTICE([ENABLE_ZMAKECERT defined])])
 
+# Check for czmq_selftest intent
+AC_ARG_ENABLE([czmq_selftest],
+    AS_HELP_STRING([--enable-czmq_selftest],
+        [Compile 'czmq_selftest' in src [default=yes].]),
+    [enable_czmq_selftest=$enableval],
+    [enable_czmq_selftest=yes])
+
+AM_CONDITIONAL([ENABLE_CZMQ_SELFTEST], [test x$enable_czmq_selftest != xno])
+AM_COND_IF([ENABLE_CZMQ_SELFTEST], [AC_MSG_NOTICE([ENABLE_CZMQ_SELFTEST defined])])
+
 # Checks for library functions.
 AC_TYPE_SIGNAL
 AC_CHECK_FUNCS(perror gettimeofday memset getifaddrs)

